optaplanner-core/src/test/resources/org/optaplanner/core/config/solver/testSolverConfig.xml

```diff
@@ -66,6 +66,7 @@
   <localSearch>
     <termination>
       <terminationCompositionStyle>AND</terminationCompositionStyle>
+      <spentLimit>PT2S</spentLimit>
```

Comment:
(hard)
This might randomly break some tests I believe, if run on very slow machines.

Reply:
The test that uses this XML does not run the solver. It just checks the (un)marshalling of the configuration.